packages/runtime/test/lib/testUtils.ts

```diff
@@ -700,12 +702,12 @@ export async function waitForRecipientToReceiveNotification(
 
 /**
  * The owner of a RelationshipAttribute receives a Request of a
- * peer and forwards them the ThirdPartyRelationshipAttribute,
+ * peer and forwards them the RelationshipAttribute,
  * waiting for all communication and event processing to finish.
  *
- * Returns the sender's own shared ThirdPartyRelationshipAttribute.
+ * Returns the sender's own shared RelationshipAttribute.
```

Comment:
Here it is actually really important to use ThirdPartyRelationshipAttribute, since with our current terms this implies that the original RelationshipAttribute is returned and not the ThirdPartyRelationshipAttribute.